game/world/objects/npc.cpp

```diff
@@ -1691,6 +1695,21 @@ bool Npc::implAiFlee(uint64_t dt) {
   return true;
   }
 
+bool Npc::useLadder() {
```

Comment:
Name `useLadder` suggests general use, but this function can really work only with go2-pathing